tox.ini

```diff
@@ -10,3 +10,6 @@ envlist = py26, py27, py34, pypy
 commands = py.test
 deps =
     pytest
+    fontTools
+    robofab
```

Comment:
Granted, the tests are also failing because ufoLib is also not on PYPI, and ufoLib requires features from fontTools that don't seem to be published to PYPI. Basically it seems like to get this test suite running at least 4 other packages need to be updated. It's gotten too frustrating for me to continue today but I'll see if I can resolve this another time. Maybe I'm not seeing this clearly and I need fresh eyes.